src/js/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -84,7 +87,7 @@ export default class ReviewCollapsibleChapter extends React.Component {
                 {hasNonArrayFields &&
                   <FormPage
                       reviewPage
-                      hideTitle={this.props.pages.length === 1}
+                      hideTitle={pages.length === 1}
```

Comment:
Since this is checking the unfiltered page list, if a chapter has two pages and one is conditional, this will show the title, even if the user only encountered one page. Shouldn't affect the 1995, but I think it's more consistent to check that the length of the active pages is 1.